<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheaterTypesForm</class>
 <widget class="QMainWindow" name="TheaterTypesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>268</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Theater Types</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/Images/theater type pic.png</normaloff>:/Images/theater type pic.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbtDelete">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTableView" name="tvwTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtLoadTable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtModify">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtAddNew">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add new</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
